### PR TITLE
Client guide revamp

### PR DESCRIPTION
**Pull request by aldesantis**
Turns the client guide into a separate section of the playbook and adds some more content, shifting the focus to the value our process brings to clients.

We'll need an illustration for the Client guide section.

## Todo

- [x] This change needs a public announcement, _I'll update [the agenda of our monthly meeting](https://tadum.app/rhythms/3JAFLu5H6JKDvLrfzXFZL4nf) once merged_.
- [x] This change moves content around, _I have configured the appropriate [Netlify redirects](https://www.netlify.com/docs/redirects/)._

### CODE REVIEW

File: client-guide/onboarding.html.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+   [Here's a sample contract](/middleman/uploads/contract-draft.pdf).
+4. **Tool onboarding:** Once the contract is signed, we will ask you to provide access to all the
+   tools we will use to work on your project. Typically, these include tools such as Slack, GitHub
+   and anything else you use to organize your work.
+5. **Team meeting:** At this point, we will set up a meeting with your new team and ask you to bring
+   any members of your team that will interact frequently with the developers. The teams will get to
+   know each other and make sure there are no blockers to starting the work. 
+6. **Development kick-off:** Once we are all set, we will start working on your project!
+
+If you need to scale your team up, we will simply repeat steps 3-6.
```
Comment (kennyadsl) [inline client-guide/onboarding.html.md]: I think our contract already covers team scaling so maybe we don't need to re-negotiate and sign the contract again.

File: client-guide/work-schedule.html.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+hours in the week.
+
+### What if I need more hours?
+
+If you are afraid that 32 hours/week are not enough to accomplish what is needed, you always have
+the option of scaling up the team.
+
+However, most clients find that, even with only four days/week, their team maintains a velocity that
+allows them to accomplish all the scheduled tasks. We suggest, at the very least, trying it out and
+seeing what works fro you.
```
Comment (kennyadsl) [inline client-guide/work-schedule.html.md]: fro -> for

File: client-guide/work-schedule.html.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+you will not only improve the quality of our work together, but also the efficiency of your entire
+team. Here are some recommendations:
+
+- **Make information available upfront.** Do not require team members to ask you for information,
+  but make all the details at your disposable available upfront where they can see them. One example
+  of this would be describing Scrum stories exhaustively before the start of the sprint, so that
+  developers can work on them without having to go through you first.
+- **Minimize synchronous communication.** Sometimes, face time is needed to deliver information in
+  the appropriate context or to reach consensus around a particularly difficult decision. Most of
+  the time, though, asynchronous communication in a Slack channel, an email thread or a Google
```
Comment (kennyadsl) [inline client-guide/work-schedule.html.md]: I'd remove the email thread. In my experience at least, it leads to a kind of communication that is not left available for the whole team and easy to be accessed anytime. WDYT?

File: client-guide/estimates-and-deadlines.html.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Closely tied to estimates is the concept of deadlines. In an ideal world, of course, deadlines would
+not be needed at all and everything would take exactly as long as it needs to take. In the real
+world, though, things are much different: circumstances outside of our control are often at play and
+sometimes we simply cannot afford to wait forever.
+
+We understand this, and we will be happy to accommodate reasonable deadlines justified by real
+business requirements. What is important is being transparent with your developers about the context
+and the needs that have led to this deadline, and why it is important to your business. This enables
+the team to be aligned with your objectives, and to suggest solutions and compromises that will
+allow you to reach them.
```
Comment (kennyadsl) [inline client-guide/estimates-and-deadlines.html.md]: What about adding an example here? Maybe a Black Friday promotion is something that fits this example nicely. Also, we could mention here that we would work hard to match the deadline but when it's not possible with a sustainable pace, we could propose an alternative that partially meets the requirements, it's well done and it's better than nothing.

File: client-guide/work-schedule.html.md (hunk 3 above)
Comment (aldesantis) [reply client-guide/work-schedule.html.md]: @kennyadsl agreed. I also mentioned that information should be public unless it absolutely needs to be private. Thanks!

File: client-guide/estimates-and-deadlines.html.md (hunk 4 above)
Comment (aldesantis) [reply client-guide/estimates-and-deadlines.html.md]: @kennyadsl done, thanks!

File: client-guide/onboarding.html.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+   project, this might be a specification document or your existing codebase.
+3. **Negotiation and contract:** If we determine that we can provide value to your project, we will
+   send you an overview of the resources we expect will be needed to accomplish the required goals,
+   along with their cost. If everything looks good to you, we will then send a contract to sign.
+   [Here's a sample contract](/middleman/uploads/contract-draft.pdf).
+4. **Tool onboarding:** Once the contract is signed, we will ask you to provide access to all the
+   tools we will use to work on your project. Typically, these include tools such as Slack, GitHub
+   and anything else you use to organize your work.
+5. **Team meeting:** At this point, we will set up a meeting with your new team and ask you to bring
+   any members of your team that will interact frequently with the developers. The teams will get to
```
Comment (davidedistefano) [inline client-guide/onboarding.html.md]: I understand the point of this sentence, but the use of the word team referring to both ours and theirs is a little bit confusing. I would rather use something like `your new dedicated team` and `any members of your organization`.

Comment (aldesantis) [reply client-guide/onboarding.html.md]: @davidedistefano thanks, fixed with "You will get to know your team"! :)

**Pull request merged**